data_diff/databases/presto.py

```diff
@@ -38,7 +37,23 @@ class Presto(Database):
     def __init__(self, **kw):
         prestodb = import_presto()
 
-        self._conn = prestodb.dbapi.connect(**kw)
+        if kw.get("schema"):
+            self.default_schema = kw.get("schema")
+
+        if (
+            "password" in kw and "auth" in kw and kw.get("auth") == "basic"
```

Comment:
If `auth == 'basic'`, the lack of user/password should throw an exception, rather than fail silently.

Reply:
Thanks, I have implemented this